### PR TITLE
Fixed syntax in Your First Dashboard

### PR DESCRIPTION
**Pull request by jomar-honsi**
Removed ordered lists, fixed minor issues.

### CODE REVIEW

**Review by karolkolodziej — APPROVED** [review]

**Review by sebastianbochan — APPROVED** [review]

**Review by ddragula — APPROVED** [review]
Thanks!

File: docs/dashboards/types-of-components.md [hunk 1]
```diff
@@ -12,37 +11,38 @@ This is an overview of the most critical parameters of a component:
...
-
-    Code snippet:
-    ```js
-    {
-        type: 'HTML',
-        renderTo: 'dashboard-1',
-        html: '<img src="https://www.highcharts.com/samples/graphics/stock-dark.svg">'
-    }
-    ```
+### **Abstract Syntax Tree (AST) style**
```
Comment (stitot) [inline docs/dashboards/types-of-components.md:18]: On a general basis we shouldn't use ** in #. Let the CSS handle formatting of headers.

File: docs/dashboards/types-of-components.md [hunk 2]
```diff
@@ -12,37 +11,38 @@ This is an overview of the most critical parameters of a component:
...
+    renderTo: 'dashboard-1',
+    elements: [{
+        tagName: 'img',
+        attributes: {
+            src: 'https://www.highcharts.com/samples/graphics/stock-dark.svg'
+        }
+    }]
+}
+```
+### **HTML string**
```
Comment (stitot) [inline docs/dashboards/types-of-components.md:34]: Remove **

**Review by stitot — CHANGES_REQUESTED** [review]
I was probably unclear on what I wanted to achieve here. My bad. I would like to keep the step by step thing, but with headers, not as a list. Se https://www.highcharts.com/docs/datagrid/general for inspiration.

Short and to the point` ## Headers`, that will provide the outline of the process in the right navigation menu.

**Review by stitot — CHANGES_REQUESTED** [review]
Great work! Small details, but big impact on clarity.

**Review by stitot — APPROVED** [review]
Thanks!

**Pull request merged**